### PR TITLE
corectl 0.7.17

### PR DESCRIPTION
**Pull request by AntonioMeireles**
hey all again!

new release :-)

### CODE REVIEW

**Comment by ilovezfs** [comment]
@AntonioMeireles was wondering when this was going to show up!

**Comment by AntonioMeireles** [comment]
@ilovezfs 

hey! btw when we'll we ge automated sierra builds in the CI ?

**Comment by ilovezfs** [comment]
@AntonioMeireles we're manually triggering the Sierra bottling here https://bot.brew.sh/job/Homebrew%20Sierra%20Testing/ but won't displace 10.9 in CI until Apple actually ships 10.12 in the App Store.

File: Formula/corectl.rb [hunk 1]
```diff
@@ -33,10 +34,7 @@ def install
 
     cd path do
       system "opam", "init", "--no-setup"
-      qcow_format_revision = build.head? ? "master" : "96db516d97b1c3ef2c7bccdac8fb6cfdcb667a04"
-      system "opam", "pin", "add", "qcow-format",
-        "https://github.com/mirage/ocaml-qcow.git##{qcow_format_revision}"
-      system "opam", "install", "uri", "qcow-format", "ocamlfind"
+      system "opam", "install", "uri", "ocamlfind", "qcow-format"
```
Comment (ilovezfs) [inline Formula/corectl.rb]: what's happened to the wonderful `opam pin` that you :heart: so much?

Comment (AntonioMeireles) [inline Formula/corectl.rb]: things got merged upstream and started being distributed in regular ocam repos. actually that pin ended lately being problematic (see https://github.com/TheNewNormal/corectl/issues/90)...

Comment (ilovezfs) [inline Formula/corectl.rb]: why not bump it to 5677d6fef5cebe03d98b854780ecc0fe99a6cb34?

Comment (AntonioMeireles) [inline Formula/corectl.rb]: consistency. it's part of stock ocam tree now. i'm not tagging uri or ocamlfind either

Comment (ilovezfs) [inline Formula/corectl.rb]: sounds fine to me then.

**Comment by zmwangx** [comment]
Merged as c2ed2f5. Thank you for your contribution to Homebrew!

Re ETA for merge: We are a small team so it's pretty hard to say.

**Pull request closed without merging**